### ISSUE TITLE
Order of tags

### ISSUE THREAD

**Issue by jonathanfield** [PTFS-Europe/rebus-list-2#24]
Call 106786 - I think tags currently display in order added. Would be nice if they displayed a-z.

**Comment by AndrewIsh** [comment]
Yes, you're right, they are currently displayed in creation/addition order. Either a syspref or user control in the UI for determining tag sort order would be relatively straightforward. I'll add a Github issue in the main RL repo.

**Comment by AndrewIsh** [comment]
https://github.com/PTFS-Europe/rebus-list/issues/825

**Comment by mrenvoize** [comment]
Also requested by Tavistock

**Comment by mrenvoize** [comment]
#93 is also a duplicate of this

**Comment by hloliver** [comment]
Hi - is there an update on this issue? I've just added some tags to an item and after saving they had resorted themselves into a completely different order, seemingly random. I added an extra tag and they shuffled themselves again after saving.

If it isn't possible to assign order then we'd prefer to preserve the order that they're added so that significant tags can entered first/last - I've seen that other sites thought a=z might be preferable, is this something that could be configurable?

Thanks
Helen

**Comment by mrenvoize** [comment]
Expected before May 31st 2017 Release
